great_expectations/expectations/expectation.py

```diff
@@ -1353,7 +1353,7 @@ def run_diagnostics(  # noqa: PLR0913
             ignore_only_for:  Bool object that when True will ignore the only_for list on Expectation sample tests.
             for_gallery:  Bool object that when True will create empty arrays to use as examples for the Expectation Diagnostics.
             debug_logger (optional[logging.Logger]):  Logger object to use for sending debug messages to.
-            only_consider_these_backends (optional[List[str]])  List of backends to consider.
+            only_consider_these_backends (optional[List[str]]): List of backends to consider.
```

Comment:
Missing colon results in weird rendering - we should double check any public API PRs to ensure the format meets our expectations